apiserver/common/package_test.go

```diff
@@ -18,6 +18,7 @@ import (
 //go:generate go run go.uber.org/mock/mockgen -typed -package mocks -destination mocks/storage.go github.com/juju/juju/state/binarystorage StorageCloser
 //go:generate go run go.uber.org/mock/mockgen -typed -package mocks -destination mocks/environs.go github.com/juju/juju/environs BootstrapEnviron
 //go:generate go run go.uber.org/mock/mockgen -typed -package mocks -destination mocks/objectstore.go github.com/juju/juju/core/objectstore ObjectStore
+//go:generate go run go.uber.org/mock/mockgen -typed -package mocks -destination mocks/facade_mock.go github.com/juju/juju/apiserver/facade WatcherRegistry
```

Comment:
While you're in there...

There are several commands here mocking interfaces from apiserver/common. Let's put them in one command, populating one file.

Reply:
Good point, changed, the line is now 504 chars .

Reply:
These mocks are a mess, I also moved them all to package_test because some generate pragmas were in the production files.